Build/jenkins/Jenkinsfile_openjdk-systemtest

```diff
@@ -0,0 +1,96 @@
+#!groovy
+pipeline {
+    agent {label "linux&&x64&&test"}
+    parameters {
+		choice (choices: 'SE90\nSE80', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
+		choice (choices: 'openjdk9-openj9\nopenjdk9\nopenjdk8', description: 'What is JVM Version?', name: 'JVM_VERSION')
+		string(defaultValue: "", description: '(Optional) To run tests individually, please provide a comma separated list of test targets from the list : {' + 
+			'ClassLoadingTest, ConcurrentLoadTest, DirectByteBufferLoadTest, ' + 
+				'LangLoadTest, LockingLoadTest, MathLoadTest_all, MathLoadTest_autosimd, ' + 
+				'MathLoadTest_bigdecimal, MauveSingleThreadLoadTest, MauveSingleInvocationLoadTest, ' + 
+				'MauveMultiThreadLoadTest, NioLoadTest, UtilLoadTest, LambdaLoadTest, JdiTest}', 
+				name: 'INDIVIDUAL_TESTS_TO_RUN')
+		 string(defaultValue: "0", description: '(Optional) Provide a variation number for the test', name: 'VARIATION_NUMBER')
+	}
+	environment {
+		PLATFORM='x64_linux'
+		MACHINE_LABEL='linux'
+        JCL_VERSION='current'
+        OPENJDK_TEST="$WORKSPACE/openjdk-test"
+        SPEC='linux_x86-64'
+        ANT_PATH="$WORKSPACE/systemtest_prereqs/apache-ant-1.10.1/bin"
+        OPENJDK_SYSTEMTEST_ROOT="$WORKSPACE/openjdk-test/openjdk-systemtest"
+        JAVA_HOME="$WORKSPACE/openjdkbinary/j2sdk-image/"
+        IS_SYSTEM_TEST=1
```

Comment:
As this variable is only used once in setup stage I would suggest move it to setup stage. Can also just remove this variable and use "1" directly.